README.md

```diff
@@ -16,19 +16,19 @@ This way you do not have to duplicate any routing information in Symfony. Consid
 [zircote/swagger-php](https://github.com/zircote/swagger-php) like the following example:
 
 ```php
-use Swagger\Annotations as SWG;
+use Openapi\Annotations as OA;
```

Comment:
the casing is wrong. the namespace is called `OpenApi`. it seems not correct in several places.